src/components/plot/plot.scss

```diff
@@ -37,12 +37,14 @@
   float: right;
 }
 
-.bookmark-button, .copy-button {
+.command {
   cursor: pointer;
-  margin-left: 4px;
+  margin-left: 20px;
```

Comment:
Why changing this to 20?   Can you share a screenshot?  In general, sharing a screenshot when you make style change is very helpful for review.  

cc: @felixcodes